spec/features/user_views_authors_list_page_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+describe "when user visit authors list page" do
+
+  it "sees authors full names, links to their pages and biography" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.